bin/cli/Cargo.toml

```
@@ -0,0 +1,19 @@
+[package]
+authors.workspace = true
+edition.workspace = true
+name              = "near-da-cli"
+version.workspace = true
+
+[dependencies]
+anyhow                = "1.0.75"
+clap                  = { version = "4.4", features = [ "derive" ] }
+hex                   = "0.4.3"
+near-da-http-api-data = { path = "../../crates/http-api-data" }
+near-da-primitives    = { path = "../../crates/primitives" }
+near-da-rpc           = { path = "../../crates/da-rpc" }
+serde                 = "1"
+serde_json            = "1.0.108"
+tokio                 = { version = "1", features = [ "full" ] }
+tower-http            = { version = "0.4", features = [ "trace" ] }
+tracing               = "0.1"
+tracing-subscriber    = { version = "0.3", features = [ "env-filter" ] }
```

```suggestion
tracing.workspace = true
tracing-subscriber.workspace = true
```

Ideally most deps that are reused should have .workspace = true, (workspace deps in the root `Cargo.toml#workspace.dependencies`